File: moz-intranet-front2/src/app/banque/detail-patrimoine/gestion-type-patrimoine/TypePatrimoine.ts
export class TypePatrimoine {

  private _id: number;
  private _name: string;


  constructor(name: string) {
    this._name = name;
  }


  get id(): number {
    return this._id;
  }

  set id(value: number) {
    this._id = value;
  }

  get name(): string {
    return this._name;
  }

  set name(value: string) {
    this._name = value;
  }

  public toJSON(){
    return {
      id: this.id,
      name: this.name
    }
  }
}

File: src/main/java/moz/outils/intranet/Banque/Entity/TypePatrimoine.java
package moz.outils.intranet.Banque.Entity;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;

@Entity
public class TypePatrimoine {

    @Id
    @GeneratedValue
    public Integer id;
    public String name;

    public TypePatrimoine() {
    }

    public TypePatrimoine(Integer id, String name) {
        this.id = id;
        this.name = name;
    }

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }
}
